helpers/fetch-issues.ts

```diff
@@ -0,0 +1,72 @@
+import fs from "fs"
+import path from "path"
+
+import { constructRepoQueries } from "../graphql/queries/get-issues"
+import { getClient } from "../lib/apollo-client"
+import projects from "../public/open-source-projects/index.json"
+import { Issue, Projects } from "../types"
+
+type ProjectIssue = {
+    url: string
+    publishedAt: string
+    title: string
+    labels: string[]
+}
+
+const labels = ["good first issue", "bug", "help wanted"]
+const states = ["OPEN"]
+
+const projectRepoMetadata = Object.entries(projects as Projects).map(
+    ([_, project]) => ({
+        name: project.name,
+        owner: project.org,
+        languages: project.lang
+    })
+)
+const repoWithLabelsAndStates = projectRepoMetadata.map((repo) => ({
+    ...repo,
+    labels,
+    states
+}))
+
+const fetchAndSaveIssues = async () => {
+    try {
+        const apolloClient = getClient()
+        const query = constructRepoQueries(repoWithLabelsAndStates)
+        console.log("Fetching issues...")
+        const { data } = await apolloClient.query({ query })
+        for (const [key, issuesData] of Object.entries(data)) {
+            if (!issuesData) continue
+            const issues = (issuesData as any)?.issues?.edges as any[]
+            const projectIssues: ProjectIssue[] = []
+            issues.map((edge: { node: Issue }) => {
+                projectIssues.push({
```

Comment:
I think you need add the repository languages and tags here, something like

`languages: projectRepoMetadata[index].languages`,
  
  `tags: projectRepoMetadata[index].tags`